src/utilities.js

```diff
@@ -35,3 +35,20 @@ export const getBabelLoader = (config, isOutsideOfApp) => {
   }
   return babelLoader;
 };
+
+export const tap = (message, options) => (config) => {
+  const print = []
+  print.push(`Tapping the configuration`)
```

Comment:
I think we should drop this non-configurable message. That's what `message` is for :)

Reply:
Done.